### PR TITLE
feat: added  ui for download csv feature

### PR DESCRIPTION
**Pull request by Ishaan28malik**
Added Download CSV feature 

`API` needs to be integrated

### CODE REVIEW

File: packages/ui/src/components/molecules/Table/TableSearchFilter.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+import { DownloadCsv } from '../../../assets';
+import { Flex, SearchBar, Tooltip } from '../../atoms';
 
 interface SearchFilterProps {
   placeholder: string;
   value: string;
   onChange: (value: string) => void;
+  handleDownloadCSV?: () => void;
+  disabled?: boolean;
+  toolTip: string;
```
Comment (irshadCypherock) [inline packages/ui/src/components/molecules/Table/TableSearchFilter.tsx]:
```suggestion
  handleDownloadCSV?: () => void;
  downloadCSVDisabled?: boolean;
  downloadCSVTooltip: string;
```

File: .changeset/five-forks-thank.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@cypherock/cysync-core': patch
+'@cypherock/cysync-ui': patch
```
Comment (irshadCypherock) [inline .changeset/five-forks-thank.md]: Add `@cypherock/cysync-desktop` to the changeset as well.

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: .changeset/five-forks-thank.md (hunk 2 above)
Comment (Ishaan28malik) [reply .changeset/five-forks-thank.md]: Oh I forgot

File: .changeset/forty-bottles-explode.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+'@cypherock/cysync-core': patch
+'@cypherock/cysync-ui': patch
+'@cypherock/cysync-desktop': patch
+---
+
+pr review changes
```
Comment (irshadCypherock) [inline .changeset/forty-bottles-explode.md]:
```suggestion
added ui for download csv history feature
```

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**